docs/rest-api/api/driveitem_get_content.md

```diff
@@ -23,7 +23,7 @@ One of the following permissions is required to call this API. To learn more, in
 <!-- { "blockType": "ignored" } -->
 
 ```http
-GET /drives/items/{item-id}/content
+GET /drives/{driver-id}/items/{item-id}/content
```

Comment:
To be more consistent with the other examples, can we swap to `GET /drive/items/{item-id}/content` instead?

Reply:
I have try to use  GET /drives/items/{item-id}/content ,it return a error code of 400,and the message is : 
```
 {
  "error": {
    "code": "BadRequest",
    "message": "Resource not found for the segment '01FASBQH4B3TCO37AHZRCLRP3CVRSN34QX'.",
    "innerError": {
      "request-id": "4c90ff6a-37df-40cd-8436-e8e93fd087de",
      "date": "2017-12-07T09:07:46"
    }
  }
}
```

But I try to use **  GET /drives/{driver-id}/items/{item-id}/content **,it return a correct response.
So maybe the right api is  ** GET /drives/{driver-id}/items/{item-id}/content **

thanks

Reply:
I've updated this to be more accurate. The /drive/items example is covered already through /me/drive/items in the list of examples, so the right fix here is what @wudeyong proposed. My only change was to rename "drives-id" to "drive-id" to match the way we reference that in other places in the content. I'll accept after the test run completes.